components/concourse-harbor-resource/.travis.yaml

```diff
@@ -0,0 +1,15 @@
+language: bash
```

Comment:
Are we still using travis?

Reply:
I think it's probably worth using shellcheck where possible ... I'll move it to the root 👍

Reply:
moved